SteakBot.Core/EventHandlers/CustomMessageHandlers/CommandMessageHandlers/CustomCommandMessageHandler.cs

```diff
@@ -22,29 +22,24 @@ protected override bool InvokeInner(SocketUserMessage message)
             var channel = message.Channel;
             var commandText = message.Content.Replace(CommandChar, "").Replace(DeleteMessageChar, "");
             var command = _memeService.MemeCommands.Single(x => x.Name == commandText);
+            var embed = new EmbedBuilder();
             switch (command.ResultType)
             {
                 case MemeResultType.Text:
-                    {
-                        channel.SendMessageAsync(command.Value).Wait();
-                        return true;
-                    }
+                    channel.SendMessageAsync(command.Value).Wait();
+                    return true;
 
                 case MemeResultType.Image:
-                    {
-                        var embed = new EmbedBuilder
-                        {
-                            ImageUrl = command.Value
-                        };
+                    embed.ImageUrl = command.Value;
```

Comment:
Maybe move the EmbedBuilder object creation in the case's body.

Reply:
Corrected.